### ISSUE TITLE
EDMC export not giving engineered modules

### ISSUE THREAD

**Issue by Pendragon55555** [EDCD/coriolis#158]
Since the API update engineered modules should be able to be exported from EDMC into coriolis, however the engineering and rolls are not being imported. It is however working with E:D Shipyard so it is definitely now a feature again.

**Comment by willyb321** [comment]
Its being worked on, we were held up by #152, also the reason it isn't working off the bat is because FDEV have changed the format of the mods.

**Comment by bazzawill** [comment]
Any update on this? I love your great work and prefer Coriolis over ED Shipyard but I am forced to use shipyard for EDMC

**Comment by willyb321** [comment]
@bazzawill Per EDCD discord:

```
Strideynet (EDStats/Coriolis) - Today at 5:57 AM
I'm afraid I'll probably have to as I've been out of the house again this weekend
I've had such a rediculous schedulre recently

**Comment by Exy-mat** [comment]
Hey, any updates on this ? Looks like EDDI doesn't work with Coriolis too, which is really frustrating

**Comment by strideynet** [comment]
Update: I'm on holiday starting next week so will have a chance to look at
it then. If you want it before then, do it yourself.

On 15 Dec 2017 9:36 pm, "Exy-mat" <notifications@github.com> wrote:

Hey, any updates on this ? Looks like EDDI doesn't work with Coriolis too,
which is really frustrating

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/EDCD/coriolis/issues/158#issuecomment-352117881>, or mute
the thread

**Comment by willyb321** [comment]
@strideynet I may actually take a poke at it in the next few days.

**Comment by willyb321** [comment]
Update: I have started work on it, check EDCD  discord for the latest news.

**Comment by willyb321** [comment]
Another update: may actually have this done in the next 12hrs

**Comment by willyb321** [comment]
I have it basically done should be ready to go to beta (hopefully) today.

/cc @strideynet

**Comment by willyb321** [comment]
Well, I ran into some issues. Mainly related to a lack of test data. If anyone wants to help - put a pastebin link with the raw data from EDMC either here or in #coriolis on EDCD (@ willyb321 in it too). I may also put up a temporary coriolis with these changes on it, which would make testing a lot easier.

**Comment by franciskong-lfasystems** [comment]
Hi,
This is a RAW data from EDMC.
https://pastebin.com/Tt1HaKNr
It should match this:
https://eddp.co/u/LmUzUWQr

**Comment by willyb321** [comment]
Fixed in #180
